owtf/webapp/src/containers/TargetsPage/style.scss

```diff
@@ -0,0 +1,28 @@
+.add-target-btn {
+  margin-top: 10px;
+}
+
+/* Spinner*/
```

Comment:
@sharmamohit123 You are gonna use this spinner at lot of place. Move this scss to global scss.